README.md

```diff
@@ -27,500 +87,318 @@ This project inspired by [Swaggo](https://github.com/swaggo/swag). Swaggo, uses
   - [Endpoints (API)](#endpoints-api)
     - [Arguments](#arguments)
 - [Contribution](#contribution)
+- [Examples](example/models/)
 
 ## Getting started
 
+0. Server Example [here](example/server.go)
+
 1. Get swagno package in your project
 
 ```sh
-go get github.com/go-swagno/swagno
-```
-
-2. Import swagno (We suggest "." import)
-
-```go
-import (
-  . "github.com/go-swagno/swagno"
-)
-```
-
-You can import without explicit period (.) like this: `import "github.com/go-swagno/swagno"` but then you have to add `swagno.` to functions, structs etc. ( `[]swagno.Endpoint` , `swagno.EndPoint` , `swagno.Params()` etc.)
-
-3. Create your endpoints (check [Endpoints](#endpoints-api)). Example:
-
-```go
-endpoints := []Endpoint{
-  EndPoint(GET, "/product", "product", Params(), nil, []models.Product{}, models.ErrorResponse{}, "Get all products", nil),
-  EndPoint(GET, "/product/{id}", "product", Params(IntParam("id", true, "")), nil, models.Product{}, models.ErrorResponse{}, "", nil),
-  EndPoint(GET, "/product/{id}/detail", "product", Params(IntParam("id", true, "")), nil, models.Product{}, models.ErrorResponse{}, "", nil),
-  EndPoint(POST, "/product", "product", Params(), models.ProductPost{}, models.Product{}, models.ErrorResponse{}, "", nil),
+go get github.com/domhoward14/swagno
```

Comment:
You will probably revert module names back but I am commenting for just a reminder